tests/contrib/hooks/test_emr_hook.py

```diff
@@ -62,7 +62,7 @@ def test_create_job_flow_extra_args(self):
         # coverage of EMR isn't 100% it turns out.
         cluster = hook.create_job_flow({'Name': 'test_cluster',
                                         'ReleaseLabel': '',
-                                        'AmiVersion': '3.2'})
+                                        'AmiVersion': '3.2', 'Instances': {'TerminationProtected': True}})
```

Comment:
Could you extend the tests here to make sure that we are actually getting a merge here